docs/reference/search/suggesters/completion-suggest.asciidoc

```diff
@@ -144,17 +146,71 @@ POST music/_suggest?pretty
 The configured weight for a suggestion is returned as `score`.
 The `text` field uses the `input` of your indexed suggestion.
 
-The basic completion suggester query supports the following two parameters:
+Suggestions are document oriented, you can specify any doc values
```

Comment:
Yes, you are right. It will work with non doc-values fields too. I have updated the docs to reflect that, but not sure if we should just encourage using doc values enabled fields only?